cmd/maintainer.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keep-network/keep-core/config"
+	"github.com/keep-network/keep-core/pkg/maintainer"
+	"github.com/spf13/cobra"
+)
+
+// MaintainerCommand contains the definition of the maintainer command-line
+// subcommand.
+var MaintainerCommand = &cobra.Command{
+	Use:   "maintainer",
+	Short: `Starts keep maintainers`,
+	Long:  `The maintainer command starts keep maintainers`,
```

Comment:
```suggestion
	Long:  `The maintainer command starts tBTC maintainer`,
```

Reply:
Done in 9fe493876ecf200cc755932defeb2498e0026d16.